src/app/components/preview-card/preview-card.component.html

```diff
@@ -0,0 +1,22 @@
+<mat-tab-group (selectedTabChange)="tabMonitor($event)" animationDuration="0ms">
+  <mat-tab *ngFor="let tabName of tabLabelList" label="{{tabName}}">
+    <mat-card *ngFor="let c of newChallenges">
```

Comment:
Not a deal breaker, but I'd prefer a more descriptive name if possible, e.g.

```ts
let challenge of newChallenges
```

Reply:
I agree! I should've used more descriptive variable names 📝 . 
Thanks

Reply:
+1

Reply:
@rrchai to address this comment.

Reply:
updated based on Verena's suggestion 513d22e